types/navigo/index.d.ts

```diff
@@ -2,6 +2,7 @@
 // Project: https://github.com/krasimir/navigo
 // Definitions by: Adrian Ehrsam <https://github.com/aersamkull>
 //                 Dancespiele <https://github.com/dancespiele>
+//                 Daniel Almaguer <https://github.com/deini>
```

Comment:
We are already two reviewers. Do you need to be reviewer too?. Waiting what think @aersamkull about it

Reply:
@dancespiele I don't need to be a reviewer, but I don't see why not. The more the better? 

Unless I'm missing something, if there are more reviewers wouldn't it make it easier to spot discrepancies on the typings such as the ones I'm updating?

Reply:
Why not, the more the better :)

Reply:
Ok then I approve